File: Programmers/Lv1/Solution_76501.java
package Lv1;

import java.util.Arrays;

public class Solution_76501 {
  public int solution(int[] absolutes, boolean[] signs) {

    for(int i = 0; i < absolutes.length; i++) {
      if(!signs[i]) absolutes[i] = -absolutes[i];
    }

    return Arrays.stream(absolutes).sum();
  }
}

File: Programmers/Lv1/Solution_76501.cpp
#include <vector>

using namespace std;

int solution(vector<int> absolutes, vector<bool> signs) {
    int answer = 0;

    for(int i = 0; i < absolutes.size(); i++) {
        int cur = signs[i] ? absolutes[i] : -absolutes[i];

        answer += cur;
    }

    return answer;
}
